### DISCUSSION TITLE
Maybe stupid questions

### DISCUSSION THREAD

**Discussion by mikensan** [TandoorRecipes/recipes#303] [General]
The simple stupid question: Is this supposed to OCR the PDF and fill out the recipe page? Or is importing/converting a PDF meant to be a manual process?

The user-error stupid question: I'm trying to configure nextcloud (self-hosted) and after trying both an app password and actual password, I continue to receive Error 500 when I sync. I suspect this may be something I did wrong (Sync location is just /recipes). If this is more a bug, then I'm happy to fill it out and provide any details.

I was able to get dropbox working - thank you for the api token link - made life easier.

**Comment by vabene1111** [comment]
so there are multiple questions here:

1. No OCR or anything, the PDFs are simply displayed on the page. This is meant to help if you have a large PDF collection and dont wont to convert all of them manually. OCR is simply to unreliable to do this, at least with the development effort i can put into it.
2. Nextcloud should work, make sure you use the webdav link and not just your instance URL, an app passwort should be the way to go.

**Reply by mikensan** [reply]
Roger that on #1.
#2 - the page mentions to not enter the webdav for nextcloud as it is appended automatically - I went ahead and tried (/remote.php/dav/files/matt) anyway for fun. No dice. https://imgur.com/a/YKG0RX6 (screenshot of the storage page). Yesterday I was running version nextcloud 19, went and updated it to 20. As for recipes - I pulled the latest version.

**Reply by vabene1111** [reply]
Oh yes i forgot that .. i will need to test nextcloud again i guess. Can you open an issue with exactly what you did and i will try to verify it

**Reply by mikensan** [reply]
Roger that will do! I just found the import websites feature, that works really freaking well. Definitely digging this.
